stock_move_line_expiration_date_required/__manifest__.py

```diff
@@ -3,12 +3,15 @@
 {
     "name": "Stock Move Line Expiration Date Required",
     "Summary": "Expiration date is required to enter manually on Move Lines.",
-    "version": "16.0.1.0.1",
+    "version": "16.0.1.0.2",
```

Comment:
**suggestion(non-blocking):** usually, you shouldn't use migration scripts in patch versions. That should indicate that the version bump should be major.

According to [our guidelines](https://github.com/OCA/odoo-community.org/blob/8afdb5a5ebfd3314e2fc836c9b9f644468ee044e/website/Contribution/CONTRIBUTING.rst#L71-L72), it'd be a major or minor bump.

Future-proofing the code, [it's better to restrict migration scripts to major or minor versions too](https://github.com/orgs/OCA/discussions/117#discussion-5699815).

However, this one PR is a little bit crossing the line because it's really a bugfix. Thus, this is not required at this time IMHO. But at least there's some extra info for you to keep in mind for next times :)